src/component/CodeBlock.tsx

```diff
@@ -1,9 +1,15 @@
 import React from "react";
-import SyntaxHighlighter, {
+import {
+  Light as SyntaxHighlighter,
   SyntaxHighlighterProps
 } from "react-syntax-highlighter";
 import lightTheme from "react-syntax-highlighter/dist/cjs/styles/hljs/atom-one-light";
 import darkTheme from "react-syntax-highlighter/dist/cjs/styles/hljs/atom-one-dark";
+import javascript from "react-syntax-highlighter/dist/cjs/languages/hljs/javascript";
+import typescript from "react-syntax-highlighter/dist/cjs/languages/hljs/typescript";
+import json from "react-syntax-highlighter/dist/cjs/languages/hljs/json";
+import markdown from "react-syntax-highlighter/dist/cjs/languages/hljs/markdown";
+import shell from "react-syntax-highlighter/dist/cjs/languages/hljs/shell";
```

Comment:
Why is this cjs?

Reply:
Otherwise tests don't work because jest + cra can't handle esm imports.